_posts/2024-03-25-whats-cooking.md

```diff
@@ -0,0 +1,224 @@
+  - [Benchmarking Bedrock Titan](#benchmarking-bedrock-titan)
+  - [Benchmarking Cohere Command](#benchmarking-cohere-command)
+  - [Benchmarking Meta Llama2](#benchmarking-meta-llama2)
+- [Conclusion (DRAFT - Jonathan)](#conclusion-draft---jonathan)
+- [Our experience](#our-experience)
+
+# Our Project (DRAFT - Febe)
+
+For our internship project, we dished out something delightful: "What's Cooking with ChatGPT."
+
+Imagine a Chrome extension that serves up AI-crafted recipes based on your online shopping list.
+
+To cook up this magic, we crafted custom web scrapers to gather data from major Belgian online retailers like Colruyt, Delhaize, Albert Heijn, and Aldi.
+
+In the backend (our Java setup), we seamlessly integrated Spring AI, opening the door to a variety of AI models, including heavy hitters like OpenAI's ChatGPT, Command by Cohere, Meta's LLama2 and Amazon's Titan.
+
+On the frontend, we spruced up the user interface with React and Next.js. Initially simple, just spitting out recipes, but soon we added a pinch of customization, letting users fine-tune preferences like dietary restrictions and culinary whims.
+
+But hold onto your chef's hat! We also added a nifty feature that conjures up a snapshot of what your dish might look like. And, of course, you can save those recipe gems for later in a convenient text file.
+
+# Used Technologies
+
+### Frontend
+
+- **Next.js:** A React framework that enables functionalities such as server-side rendering and generating static websites, enhancing SEO, performance, and user experience.
+
+- **Chakra UI:** A simple, modular and accessible component library that provides the building blocks to build React applications with speed and ease.
+
+### Backend
+
+- **Spring Boot:** Simplifies the development of new Spring applications through convention over configuration, offering a wide range of functionalities for modern web services.
+
+- **Spring AI:** Spring AI is a framework for building AI applications in Java, inspired by Spring principles for ease of development.
+
+- **LLM (Large Language Models):** Offers advanced capabilities in natural language processing, understanding, generation, and translation, enabling sophisticated interaction and content creation.
+
+### AWS Services
+
+- **ECR (Elastic Container Registry):** Provides a Docker container registry that makes it easy for developers to store, manage, and deploy Docker container images.
+
+- **ECS (Elastic Container Service):** A fully managed container orchestration service that makes it easy to run, stop, and manage containers on a cluster.
+
+- **App Runner:** Provides a managed service for developers to quickly deploy containerized web applications and APIs, without dealing with infrastructure management.
+
+- **Secret Manager:** Helps to protect access to applications, services, and IT resources without the upfront cost and complexity of managing a full infrastructure.
```

Comment:
De beschrijving hiervan is voor mij wat onduidelijk. Ik zou eerder iets zeggen als:

```suggestion
- **Secrets Manager:** Allows you to easily store and manage confidential information, such as passwords and access keys.
```